openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/domain/encounter/OpenERPOrders.java

```diff
@@ -13,7 +12,19 @@ public OpenERPOrders(String id){
     }
 
     public List<OpenERPOrder> getOpenERPOrders() {
-        return openERPOrders;
+        return removeDuplicateOrders(openERPOrders);
+    }
+
```

Comment:
Please add a comment on this method as to WHY this method exists? (what is the side effect of not calling removeDuplicateOrders).